src/assets/images/KebabMenu.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-import Svg, {Circle, G} from 'react-native-svg';
+import Svg, {Ellipse, G} from 'react-native-svg';
 
 /* tslint:disable */
 
 const KebabMenu: React.FC<any> = ({fill, width = 32, height = 32}) => (
 	<Svg width={width} height={height} viewBox='0 0 512 512'>
 		<G>
 			<G>
-				<Circle fill={fill} cx="256" cy="256" r="64"/>
-				<Circle fill={fill} cx="256" cy="448" r="64"/>
-				<Circle fill={fill} cx="256" cy="64" r="64"/>
+				<Ellipse fill={fill} x="256" y="256" rx="64" ry="64"/>

```

Comment:
why G here?

Reply:
more save way, i leave it as is

Reply:
G makes group of elements. Why double G?